src/hooks/useCourtContracts.js

```diff
@@ -564,44 +566,56 @@ export function useActiveBalanceOfAt(juror, termId) {
   return [activeBalance.amount, activeBalance.error]
 }
 
-export function useTotalActiveBalancePolling(termId) {
-  const jurorRegistryContract = useCourtContract(
-    CourtModuleType.JurorsRegistry,
-    jurorRegistryAbi
-  )
-  const [totalActiveBalance, setTotalActiveBalance] = useState(bigNum(-1))
+export function useTotalANTStakedPolling(timeout = 1000) {
+  const [totalANTStaked, setTotalANTStaked] = useState(bigNum(-1))
+  const [error, setError] = useState(false)
+  const [controlledTimeout, setControlledTimeout] = useState(0)
```

Comment:
Here we could use `useRef(0)` instead and update it as `controlledTimeout.current = timeout`. This way we don't do an unnecessary re-render the first time we get the total staked.